src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/utils.ts

```diff
@@ -53,7 +53,7 @@ export const THORCHAIN_SAVERS_DUST_THRESHOLDS_CRYPTO_BASE_UNIT = {
   [ethAssetId]: '10000000000',
   [avalancheAssetId]: '10000000000',
   [bscAssetId]: '10000000000',
-  [cosmosAssetId]: '0',
+  [cosmosAssetId]: '1',
```

Comment:
do we have relevant info on why this would differ from the dust limit reported by `/inbound_addresses`?

Reply:
@0xApotheosis with more info to chime here - 0 Txs indeed worked for savers so they should too?

Reply:
But that's a fair point abount `inbound_addresses` having `dust_threshold` which we may want to use instead. Though I believe the reason it is 0 for EVM chains is because it's assuming a `depositWithExpiry` calls, and for savers we sometimes do regular sends instead of a contract call, which may require some dust

Reply:
The docs do not call out Cosmos as needing a dust amount, but THORChain was not picking up these transactions without it.

This was a trial and error thing - once I added the smallest base unit amount as dust Cosmos withdrawals started working.

Reply:
would be worth adding this as a code commnet

Reply:
👌 https://github.com/shapeshift/web/pull/6285/commits/eff7f2d2ba392c71b6929ff8bd72efe433393fa5